content/02.introduction.md

```diff
@@ -9,12 +9,20 @@ The low survival rates for some pediatric tumors are clearly multifactorial, exp
+This regulatory change is poised to accelerate identification of novel and effective therapeutic for pediatric diseases with putative molecular targets that have otherwise been overlooked.
 
-Although there has been significant progress in recent years to elucidate the landscape of somatic variation responsible for pediatric brain tumor formation and progression, translation of therapeutic agents to phase II or III clinical trials and subsequent FDA approvals have not kept pace. 
-Within the last 20 years, the FDA has approved only five drugs for the treatment of pediatric brain tumors: mTOR inhibitor, everolimus, for subependymal giant cell astrocytoma [@doi:10.1097/01.COT.0000421359.88140.a8; @url:https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-everolimus-tuberous-sclerosis-complex-associated-partial-onset-seizures]; anti-PD-1 immunotherapy, pembrolizumab, for microsatellite instability–high or mismatch repair–deficient tumors [@url:https://www.fda.gov/drugs/drug-approvals-and-databases/fda-approves-pembrolizumab-adults-and-children-tmb-h-solid-tumors]; NTRK inhibitors larotrectinib [@url:https://www.fda.gov/drugs/fda-approves-larotrectinib-solid-tumors-ntrk-gene-fusions] and entrectinib [@url:https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-entrectinib-ntrk-solid-tumors-and-ros-1-nsclc] for tumors with an NTRK 1/2/3 gene fusions; MEK1/2 inhibitor, selumetinib, for neurofibromatosis type 1 (NF1) and symptomatic, inoperable plexiform neurofibromas [@url:https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-selumetinib-neurofibromatosis-type-1-symptomatic-inoperable-plexiform-neurofibromas]. 
-This is, in part, due to pharmaceutical company priorities and/or concerns regarding toxicity leading, making it challenging for researchers to obtain to new therapeutic agents for pediatric clinical trials.
-Critically, as of August 18, 2020, an amendment to the Pediatric Research Equity Act called the "Research to Accelerate Cures and Equity (RACE) for Children Act" mandates that all new adult oncology drugs also be tested in children when the molecular targets are relevant to a particular childhood cancer. 
-This regulatory change is poised to accelerate identification of novel and effective therapeutic for pediatric diseases that have otherwise been overlooked.
+To leverage diverse scientific and analytical expertise to analyze the PBTA data, we created an open science model and incorporated features, such as analytical code review [@doi:10.1093/aje/kwab092; @doi:10.7287/peerj.preprints.3210v1] and continuous integration to test data and code [@doi:10.1038/nbt.3780; @doi:10.7287/peerj.preprints.3210v1], to improve reproducibility throughout the life cycle of our project, termed OpenPBTA.
```

Comment:
I was not really aware of anything explicitly talking about _just_ using CI. But I've added some citations for related ideas, along with "to test data and code." 

From [Parker 2017](https://doi.org/10.7287/peerj.preprints.3210v1), on testing your data:

> The same can be said for data testing. Almost always, by the time a data analyst
> has gotten to the point of performing an analysis, she has made the assumption
> that the data are not corrupted, usually by inspecting the data or a subset of the
> data manually. A negative value for a weight measurement, for example, would
> immediately signal to the analyst that she should figure out what “went wrong”.
> Assertive testing means establishing these quality-control checks – usually based
> on past knowledge of possible corruptions of the data – and halting an analysis
> if the quality-control checks are not passed, so the analyst can investigate and
> hopefully fix (or at least account for) the problem. This could take the form
> of throwing an error in the analysis script, or displaying an error within the
> analysis document.

That's _part of_ what we've set up is doing for sure, but not the complete picture. When we cut a new release and therefore create new testing data, it does let us check for any regressions in the data that are able to be caught in CI (and I feel like that's been our experience in practice with earlier releases).

Then there's the concept of "continuous analysis" in [Beaulieu-Jones and Greene](https://doi.org/10.1038%2Fnbt.3780), but we're not going _quite that far_. The underlying infrastructure (i.e., continuous integration) is shared between that concept and what we've done; both serve as tests to make sure the code still runs on the data for the project.